src/css.js

```diff
@@ -58,13 +58,31 @@ function vendorPropName( name ) {
 	}
 }
 
+// Use font-size to convert to px
+function convertToPx( elem, value ) {
+	var fontSize = jQuery.css( elem, "font-size" );
+	elem.style.fontSize = value;
+	value = parseFloat( jQuery.css( elem, "font-size" ) );
```

Comment:
This will force a synchronous layout, right? I suppose there's no way to avoid it but I wonder if there is a less expensive way to do it, for example with a different element attached to the body.